Source/WebCore/platform/graphics/filters/FEDropShadow.h

```diff
@@ -51,6 +51,10 @@ class FEDropShadow : public FilterEffect {
 
     static IntOutsets calculateOutsets(const FloatSize& offset, const FloatSize& stdDeviation);
 
+#if !USE(CG)
+    void setOperatingColorSpace(const DestinationColorSpace&) override { }
+#endif
```

Comment:
This change I added after analyzing the reason of the failing LayoutTests: 
* svg/filters/repeated-drop-shadow-effects.svg
* imported/blink/svg/filters/feDropShadow-linearrgb-flood-color.svg

```
<svg width="404" height="404" xmlns="http://www.w3.org/2000/svg">
    <defs>
        <filter id="repeated-drop-shadow" filterUnits="userSpaceOnUse"/>
    </defs>
    <rect x="200" y="200" width="4" height="4" style="fill:green; filter:url(#repeated-drop-shadow);"/>
    <script><![CDATA[
        let nameSpace = "http://www.w3.org/2000/svg";
        let filterElement = document.getElementById("repeated-drop-shadow");

        function createDropShadowEffects(count, dx, dy) {
            for (let i = 0; i < count; i++) {
                var dropShadowElement = document.createElementNS(nameSpace, "feDropShadow");
                dropShadowElement.setAttribute("dx", dx.toString());
                dropShadowElement.setAttribute("dy", dy.toString());
                dropShadowElement.setAttribute("stdDeviation", "0");
                dropShadowElement.setAttribute("flood-color", "green");
                filterElement.appendChild(dropShadowElement);
            }
        }

        createDropShadowEffects(50, -2,  2);
        createDropShadowEffects(50,  2, -2);
        createDropShadowEffects(50,  2,  2);
        createDropShadowEffects(50, -2, -2);
    ]]></script>
</svg>
```

So in this case we have (without this `void setOperatingColorSpace(const DestinationColorSpace&) override { }`):
* sourceImage is in linearRGB
* SourceGraphic is in linearRGB
* FEDropShadow operates in linearRGB but the `flood-color` is in sRGB(0,128,0,255) - green
* FEDropShadow output is in linearRGB (but with color values from the sRGB(0,128,0,255) - green)
* the final image should be in sRGB(deviceRGB) so the FEDropShadow output is transformed linearRGB->sRGB and the final image has pixels with values (0, 188, 0, 255) - brighten green

The green color in linearRGB(0, 55, 0, 255).
The green color in sRGB(0, 128, 0, 255).

So to solve this problem we have two options:
1. force DropShadow to operate in sRGB color space so the same as `flood-color` argument of this filer effect
2. transform `flood-color` (argument of the filter effect) to operating color space of FEDropShadow (which depends on color-interpolation-filter argument value - default `linearRGB`)

I chose option 1 but I think to be compliance with spec we should implement option 2.

@nikolaszimmermann , @shallawa : WDYT?

Reply:
I have checked how it is in Chromium and I found that `flood-color` is transformed to operating interpolation space: 
https://chromium.googlesource.com/chromium/src/+/refs/heads/main/third_party/blink/renderer/platform/graphics/filters/fe_drop_shadow.cc#76

Reply:
It seems that transforming a `shadow-color` for `DropShadow` filter is not a good option for Cairo because before the Color is passed to Cairo it is transformed to SRGB (so transformation in `FEDropShadowSoftwareApplier::apply` does not make sense):
https://github.com/WebKit/WebKit/blob/main/Source/WebCore/platform/graphics/cairo/CairoUtilities.cpp#L118

So I propose a compromise and force SRGB operating color space in case of Cairo and DropShadow